include/boost/http_proto/rfc/detail/rules.hpp

```diff
@@ -224,7 +239,7 @@ status_line_rule =
         status_code_rule,
         grammar::squelch(
             grammar::delim_rule(' ') ),
-        grammar::token_rule(ws_vchars),
+        reason_phrase_rule,
```

Comment:
@vinniefalco any comment on this?
I could use `grammar::optional(grammar::token_rule(ws_vchars))` instead I guess.

Reply:
I'd like all that stuff to be private